overloaded_intrinsic_funcs/10_vector_permutation_functions.md

```diff
@@ -7,365 +7,365 @@
 **Prototypes:**
 ``` C
 int8_t vmv_x (vint8mf8_t src);
-vint8mf8_t vmv_s (vint8mf8_t dest, int8_t src, size_t vl);
+vint8mf8_t vmv_s (vint8mf8_t dest, int8_t src, size_t vl, uint8_t tu);
```

Comment:
Isn't this instruction's tail policy already covered by dest being vundefined or not?